DSharpPlus/Entities/Guild/DiscordMember.cs

```diff
@@ -195,6 +196,21 @@ public override string Discriminator
             internal set => this.User.Discriminator = value;
         }
 
+
+        [JsonProperty("avatar")]
```

Comment:
This should be added to `TransportMember` not `DiscordMember`

Reply:
Fix'd